### PR TITLE
Release 2020 03 10

### PR DESCRIPTION
**Pull request by dingmeng-xue**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

File: ChangeLog.md [hunk 1]
```diff
@@ -1,3 +1,86 @@
...
+#### Az.Storage
+* Supported AllowProtectedAppendWrite in ImmutabilityPolicy
+    - 'Set-AzRmStorageContainerImmutabilityPolicy'
+* Added breaking change warning message for AzureStorageTable type change in a future release
+    - 'New-AzStorageTable'
+    - 'Get-AzStorageTable'
+
+#### Az.Websites
+* Added Tag parameter for 'New-AzAppServicePlan' and 'Set-AzAppServicePlan'
+* Stop cmdlt execution if an exception is thrown when adding a custom domain to a website
```
Comment (isra-fel) [inline ChangeLog.md]: a minor typo
```suggestion
* Stop cmdlet execution if an exception is thrown when adding a custom domain to a website
```

Comment (dingmeng-xue) [reply ChangeLog.md]: I will fix when it will be merged into master. No time to resolve it.

File: tools/Az/Az.psd1 [hunk 2]
```diff
@@ -31,8 +31,7 @@ Copyright = 'Microsoft Corporation. All rights reserved.'
 
 # Description of the functionality provided by this module
 Description = 'Microsoft Azure PowerShell - Cmdlets to manage resources in Azure. This module is compatible with WindowsPowerShell and PowerShell Core.
-
-For more information about the Az module, please visit the following: https://docs.microsoft.com/en-us/powershell/azure/'
+    For more information about the Az module, please visit the following: https://docs.microsoft.com/en-us/powershell/azure/'
```
Comment (isra-fel) [inline tools/Az/Az.psd1]: Is this change intentional?

File: tools/Az/Az.psd1 [hunk 3]
```diff
@@ -162,68 +161,88 @@ PrivateData = @{
         # IconUri = ''
 
         # ReleaseNotes of this module
-        ReleaseNotes = '3.5.0 - February 2020
+        ReleaseNotes = '3.7.0 - March 2020
```
Comment (isra-fel) [inline tools/Az/Az.psd1]:
```suggestion
        ReleaseNotes = '3.6.0 - March 2020
```
